molecule/driver/docker.py

```diff
@@ -90,7 +91,7 @@ class Docker(base.Base):
             buildargs:
                 http_proxy: http://proxy.example.com:8080/
 
-    If specifying the `CMD`_ directive in your ``Dockerfile.j2`` or consuming a
+    If specifying the `CMD`_ directive in your ``dockerfile`` or consuming a
```

Comment:
That change is only to make the documentation correct. If I would use Dockerfile.j2 and user had defined a different value for dockerfile, the doc will not be correct. If you want I could mention that default value is `Dockerfile.j2`  by adding `(defaults to Dockerfile.j2)`.

Reply:
> That change is only to make the documentation correct.

Not as I understand it.

This section is about `command` and `override_command` configuration, not `dockerfile`, so it feels strange to mention it here.  And I think we should default to `Dockerfile.j2` when talking about the file because that it what users are most familiar with. Otherwise, by this logic, we should be updating multiple places in the documentation with `Dockerfile.j2` -> `dockerfile`.

Reply:
Nevermind, i removed it.